packages/storybook/stories/ControlBar/Buttons/Devices/snippets/DevicesButtonWithKnobs.snippet.tsx

```diff
@@ -5,7 +5,7 @@ import { DevicesButton, DevicesButtonProps } from '@azure/communication-react';
 import React, { useState } from 'react';
 import { defaultControlsCameras, defaultControlsMicrophones, defaultControlsSpeakers } from '../../../../controlsUtils';
 
-export const DevicesButtonWithKnobs = (args: any): JSX.Element => {
+export const DevicesButtonWithKnobs = (args: unknown): JSX.Element => {
```

Comment:
curious about this change

Reply:
We shouldn't use `any` types because you can do `myVar.foo` and typescript won't complain. If we don't know the type we should use `unknown`. Doing myVar.foo will then throw an error, so to use myVar we should explicit casing (e.g. `(myVar as KnownType).foo`.

But now I'm looking at this again, this should be `DeviceButtonProps` and the caller should do any `any->unknown->type` casting. I'll update this

Reply:
updated.